source/apps/change-deployment-models.txt

```diff
@@ -0,0 +1,208 @@
+.. _change-deployment-models:
+
+========================
+Change Deployment Models
+========================
+
+.. contents:: On this page
+   :depth: 2
+
+Overview
+--------
+
+You can change an existing App's deployment configuration to deploy it
+in a new model, region, and/or cloud provider. For a list of all
+available options, see :ref:`Deployment Models & Regions
+<deployment-model>`.
```

Comment:
can you add a short paragraph or list here about why one might want to change deployment model?